### ISSUE TITLE
Expose Controller/Dataflow Status

### ISSUE THREAD

**Issue by dwlbnks** [readysettech/readyset#264]
**Description**

Create a status command to query information on the dataflow state including

* Total migrations
* Ongoing migrations
* Last successful migration
* Last migration error
* Total Controller requests
* Last Controller request

Investigate whether there is additional information in the Registry that would be useful for debugging.

**Change in user-visible behavior**

Users will be able to query additional information about the dataflow state and controller

**Requires documentation change**

Yes

**Comment by jasobrown-rs** [comment]
@gvsg-rs should this information be added to the existing`SHOW READYSET STATUS`?  That command currently has replication-related information, and [REA-3321](https://linear.app/readyset/issue/REA-3321/expose-adapter-metrics-via-status-commands) adds adapter data - roughly 12 items total now.

**Comment by jasobrown-rs** [comment]
We've aligned internally on a pattern of `SHOW READYSET STATUS <command>` for area-specific data. Command name for this ticket TBD

**Comment by gvsg-rs** [comment]
Is this ticket completed?

**Comment by jasobrown-rs** [comment]
No, it was deferred this week due to the discourse testing, and now REA-3540
